package.json

```
@@ -86,6 +90,8 @@
     "clsx": "2",
     "commander": "8",
     "core-js": "^3.6.5",
+    "ember-template-recast": "^6.1.4",
+    "ember-template-tag": "^2.3.16",
```

```suggestion
    "content-tag": "^2.0.0",
```

`content-tag` is the official implementation of `<template>` parsing (and has types!)

See here: https://github.com/embroider-build/content-tag/

---

Resolved!